src/libxtp/internalcoords.cc

```diff
@@ -0,0 +1,683 @@
+#include <algorithm>
+#include <boost/graph/connected_components.hpp>
+#include <cmath>
+#include <iostream>
+#include <limits>
+#include <random>
+#include <votca/xtp/CoordContainer.h>
+#include <votca/xtp/internalcoords.h>
+
+namespace votca {
+namespace xtp {
+
+const std::vector<std::string> electroNegElements{"N", "O", "F",
```

Comment:
This does not appear to be an exhaustive list what is it used for? And why not put it in the votca::tools::elements?